device.mk

```diff
@@ -205,4 +205,26 @@ PRODUCT_PROPERTY_OVERRIDES += \
 #
 PRODUCT_DEFAULT_PROPERTY_OVERRIDES += \
     persist.sys.usb.config=mtp \
-    persist.data.qmi.adb_logmask=0
+    persist.data.qmi.adb_logmask=0 \
+    ro.adb.secure=0
+
+# for Gecko to support bluedroid stack
+PRODUCT_PACKAGES += \
+    bluetooth.default
+
+ENABLE_LIBRECOVERY := true
+
+# Enable virtual home button for b2g
+PRODUCT_PROPERTY_OVERRIDES += \
+    ro.moz.has_home_button=0 \
+    ro.moz.nfc.enabled=true \
+    ro.moz.ril.ipv6=true \
+    ro.moz.ril.signal_extra_int=true \
+
+PRODUCT_PACKAGES += \
+    nfcd \
+    init.sh
```

Comment:
ENABLE_LIBRECOVERY is for librecovery, but I am not sure if we need timekeep for now. We can add it later in case of need.